scripts/zones/The_Sanctuary_of_ZiTah/Zone.lua

```diff
@@ -4,71 +4,58 @@
 --
 -----------------------------------
 package.loaded["scripts/zones/The_Sanctuary_of_ZiTah/TextIDs"] = nil;
-package.loaded["scripts/globals/chocobo_digging"] = nil;
 -----------------------------------
-
 require("scripts/zones/The_Sanctuary_of_ZiTah/TextIDs");
+require("scripts/zones/The_Sanctuary_of_ZiTah/MobIDs");
 require("scripts/globals/icanheararainbow");
-require("scripts/globals/zone");
-require("scripts/globals/conquest");
 require("scripts/globals/chocobo_digging");
+require("scripts/globals/conquest");
+require("scripts/globals/missions");
+require("scripts/globals/zone");
 
 -----------------------------------
 -- Chocobo Digging vars
 -----------------------------------
 local itemMap = {
```

Comment:
as long as your are doing this and making my format cleanup get a conflict here, put the brace on newline too.